Library/Homebrew/cmd/doctor.rb

```diff
@@ -914,12 +906,11 @@ def check_git_newline_settings
   end
 
   def check_git_origin
-    return unless git? && (HOMEBREW_REPOSITORY/".git").exist?
+    return unless Utils.git_available? && (HOMEBREW_REPOSITORY/".git").exist?
```

Comment:
Just a nit (that you didn't do): I always find `unless ... &&` hard to parse compared to `if ! ...`